lib/libc/aarch64/gen/_setjmp.S

```diff
@@ -81,9 +81,7 @@ ENTRY(_longjmp)
 
 	/* Restore the stack pointer */
 	ldr	REG(8), [REG(0)], #(REG_WIDTH)
-#ifdef CHERI_LIB_C18N
-	mov	c2, c8
-#else
+#ifndef CHERI_LIB_C18N
 	mov	REGN(sp), REG(8)
```

Comment:
What's wrong with doing this for c18n? I'd like to see as much of libc etc de-ifdef'ed as eventually I'd like to see lib/c18n killed off, just as libunwind doesn't need two copies.